packages/tempus/src/Tempus_StepperBackwardEuler_impl.hpp

```diff
@@ -120,6 +120,24 @@ void StepperBackwardEuler<Scalar>::setAppAction(
 }
 
 
+template<class Scalar>
+void StepperBackwardEuler<Scalar>::setModel(
+  const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel)
+{
+  StepperImplicit<Scalar>::setModel(appModel);
+
+  if (predictorStepper_ != Teuchos::null) {
+    // If predictor's model is not set, set it to the stepper model.
```

Comment:
provide warning?

Reply:
The same as above.  Probably no warning here.